tools/unjson_logs

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Handle truncated log lines which are no longer valid JSON
+jq -c -R 'if endswith("...") then "\(.)\"}" else . end | fromjson? | .' | \
```

Comment:
Maybe we should check `which jq` and exit 1 with a message about installing it via your package manager if it's missing.

Reply:
Oh yeah that's a good idea

Reply:
Sounds good to me.